app/controllers/katello/api/rhsm/candlepin_proxies_controller.rb

```diff
@@ -333,28 +343,36 @@ def find_hypervisor_organization
       deny_access unless @organization
     end
 
-    def find_organization
+    def find_organization(key = :organization_id)
```

Comment:
since the same logic is desired no matter what param for used for organization specification, we started to accept the param name, therefore we can use it also for `params[:owner]`, to keep it backwards compatible, the key fallbacks to :organization_id